src/swap/central/exolix.ts

```diff
@@ -167,19 +168,21 @@ export function makeExolixPlugin(opts: EdgeCorePluginOptions): EdgeSwapPlugin {
           headers
         })
       }
-
-      if (
-        !response.ok &&
-        !(await response.text()).includes(
-          'Amount to exchange is below the possible min amount to exchange'
-        ) // HACK: Exolix inconsistently returns a !ok response for a 'from' quote
-        // under minimum amount, while the status is OK for a 'to' quote under
-        // minimum amount.
-        // Handle this inconsistency and ensure parse the proper under min error
-        // and we don't exit early with the wrong 'unsupported' error message.
-      ) {
-        log.warn(`Error retrieving Exolix quote: ${await response.text()}`)
+      opts.log.warn('response', JSON.stringify(response))
```

Comment:
Is this leftover debugging code, or do we want it permanently?

Reply:
definitely debugging, will remove